.github/workflows/ci.yml

```
@@ -54,4 +54,4 @@ jobs:
           pytest --cov=removestar . -vv
 
       - name: Upload coverage report
-        uses: codecov/codecov-action@v3.1.4
+        uses: codecov/codecov-action@v5.0.7
```

It is possible to revert this change and allow others to go ahead.
```suggestion
        uses: codecov/codecov-action@v3.1.4
```